### PR TITLE
Support for decimal and signed numeric keyboard

### PR DESCRIPTION
**Pull request by sbaranov**
Fixes https://github.com/flutter/flutter/issues/14916.
Requires https://github.com/flutter/flutter/pull/15846.

### CODE REVIEW

**Comment by Hixie** [comment]
Assuming the Java and ObjC code is syntactically correct, LGTM.

Can you think of a way we could test this?

**Comment by xster** [comment]
LGTM

File: shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm [hunk 1]
```diff
@@ -9,13 +9,17 @@
 static const char _kTextAffinityDownstream[] = "TextAffinity.downstream";
 static const char _kTextAffinityUpstream[] = "TextAffinity.upstream";
 
-static UIKeyboardType ToUIKeyboardType(NSString* inputType) {
+static UIKeyboardType ToUIKeyboardType(NSDictionary* type) {
```
Comment (cbracken) [inline shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm]: Optional: Maybe slightly cleaner to break out (NSString *inputType, BOOL signed)?

**Comment by cbracken** [comment]
lgtm

**Pull request merged**